app/views/issues/_table.html.erb

```diff
@@ -1,4 +1,4 @@
-<% cache ['issues-table', @columns, issues.map(&:id), @issues.map(&:updated_at).map(&:to_i).sort.last] do %>
+<% cache ['issues-table', @columns, issues.map(&:id), issues.map(&:state), @issues.map(&:updated_at).map(&:to_i).sort.last] do %>
```

Comment:
why do we need this in the key, and under what conditions would this part of the cache key change?

Reply:
Removed this one and the other cache key update. Apparently, `update_all` (in states_controller) does not call any callbacks and does not _touch_ the records. Manually updating the `updated_at` attribute in the `update_all` call fixes the issue.